db/fixtures/chargeback_rates.yml

```diff
@@ -17,6 +17,18 @@
           :finish: Infinity
           :fixed_rate: 0.0
           :variable_rate: 0.02
+    - :description: Used CPU Cores
+      :group: cpu
+      :source: used
+      :metric: cpu_usage_rate_average
```

Comment:
Is that the right metric for CPU cores? Seems like that one is just the amount of CPU mhz used. Unless you're planning on adding a calculation in a method in the Chargeback class that uses it. Either way, you'll need to extend the list in`set_columns_hash` in the class to be able to report on it. as @amaurygonzalez mentioned.